lib/mayaUsd/fileio/registryHelper.cpp

```diff
@@ -189,7 +189,6 @@ void UsdMaya_RegistryHelper::FindAndLoadMayaPlug(
                 // already loaded.
                 plug->Load();
             }
-            break;
```

Comment:
This code would stop at the first usdPreviewSurface translator found. Since we can export to multiple rendering contexts, we need to continue until all usdPreviewSurface translators are found.